content/news/2013-06-03-lightning-talks.md

```diff
@@ -4,6 +4,7 @@ Event: 2013-06-11 7:30 pm to 9:00 pm
 Location: O'Reilly Media
 Date: 2013-06-03 11:07
 Drupal_Node: 192
+Slug: lightning-talk-2013
```

Comment:
`make devserver` was complaining. Let me try and remove the slug and see if it still complains.